source/globalCommands.py

```diff
@@ -622,6 +622,23 @@ def script_toggleReportLineIndentation(self, gesture: inputCore.InputGesture):
 		# {mode} will be replaced with the mode; i.e. Off, Speech, Tones or Both Speech and Tones.
 		ui.message(_("Report line indentation {mode}").format(mode=state.displayString))
 
+	@script(
+		# Translators: Input help mode message for toggle ignore blank lines for line indentation reporting command.
+		description=_("Toggles on and off the ignoring of blank lines for line indentation reporting"),
+		category=SCRCAT_DOCUMENTFORMATTING
+	)
+	def script_toggleignoreBlankLinesForReportLineIndentation(self, gesture: inputCore.InputGesture) -> None:

```

Comment:
Fixup CamelCase for `toggleIgnore`

```suggestion
	def script_toggleIgnoreBlankLinesForReportLineIndentation(self, gesture: inputCore.InputGesture) -> None:
```